arangod/Graph/GraphManager.cpp

```diff
@@ -765,11 +767,76 @@ Result GraphManager::readGraphByQuery(velocypack::Builder& builder,
         << "cannot read graphs from _graphs collection";
   }
 
-  builder.add(VPackValue(VPackValueType::Object));
-  builder.add("graphs", graphsSlice);
+  builder.openObject();
+  if (!onlyHash) {
+    /*
+     * Will create {"graphs": [...]}
+     */
+    builder.add(VPackValue(StaticStrings::GraphsArray));
+    {
+      VPackArrayBuilder graphsArray(&builder);
+      for (VPackSlice graphSlice : VPackArrayIterator(graphsSlice)) {
+        builder.openObject();
+        auto res = writeGraphToBuilder(builder, graphSlice, details);
+        if (res.fail()) {
+          builder.close();
+          return res;
+        }
+        builder.close();
+      }
+    }
+  } else {
+    /*
+     * Will create {"checksum": "<md5-of-graphs>"}
```

Comment:
```suggestion
     * Will create {"checksum": "<checksum-of-graphs>"}
```
this is no md5 anymore